rskj-core/src/test/java/org/ethereum/datasource/KeyValueDataSourceTest.java

```
@@ -53,14 +53,34 @@ public void setup() {
 
     @Test
     public void put() {
-        byte[] randomKey = TestUtils.randomBytes(20);
-        byte[] randomValue = TestUtils.randomBytes(20);
+        byte[] randomKey = "testing-key".getBytes();
+        byte[] randomValue = "testing-value".getBytes();
 
         keyValueDataSource.put(randomKey, randomValue);
         if (withFlush) {
             keyValueDataSource.flush();
         }
         assertThat(keyValueDataSource.get(randomKey), is(randomValue));
+
+        try (DataSourceKeyIterator iterator = keyValueDataSource.keyIterator()) {
+
+            iterator.seekToFirst();
+
+            assertTrue(iterator.hasNext());
+
+            byte[] expectedValue = null;
+
+            while (iterator.hasNext()) {
+                expectedValue = iterator.next();
+                if (ByteUtil.wrap(expectedValue).equals(ByteUtil.wrap(randomKey))) {
+                    break;
+                }
+            }
+
+            assertArrayEquals(expectedValue, randomKey);
+        } catch (Exception e) {
+            e.printStackTrace();
```

I wouldn't silently ignore this exception..